### ISSUE TITLE
Name change

### ISSUE THREAD

**Issue by dbell154** [AndBible/and-bible#1758]
Name change has been discussed extensively before, namely deciding between And Bible, Bible Study, Study Bible, etc. The app store titling works with And Bible being mentioned. But I have a different perspective I would like to posit.

I was searching the web for available information about And Bible, specifically tutorials. Unfortunately in every search the results include many generic uses of the terms "and" and "Bible", things like "prayer and Bible study". Using "And Bible" or "AndBible" didn't help. Searching for Bible Study, Study Bible App, etc. didn't help. I'm thinking that the easier it is for people to find information about the app, the more helpful it will be.

Should we discuss the possibility of giving it a distinguishing name with a term less generic than "and"?
I'm thinking that the easier it is for people to find information about the app, the more helpful it will be.

**Comment by agrogers** [comment]
Hi Dana

I probably agree with you mostly on this. 'And Bible' can get some good search hits but it is definitely open to misunderstanding. I think 'AndBible' would work better but we have not used that much so it is not going work well at the moment. The 'Bible Study App' was settled on because it puts the word Bible at the beginning of the name which is important i you want 'Bible' to show in the launcher. But I agree, it is generic and so is less likely to get good hits when searching for it.  "And And Bible is getting more popular" -  which is great but the beginning of this sentence just looks really weird to me :)

I don't think there will be any changes to this going forward but good to hear your thoughts.

**Comment by tuomas2** [comment]
Name change reasons have been addressed here: https://github.com/AndBible/and-bible/issues/573

**Comment by dbell154** [comment]
I agree with the logic in selecting name(s), although it doesn't distinguish this Bible Study App from other apps described as Bible study apps when doing a web search such as The Study Bible App from Grace to You. The current name does work well in the Google Play search, so any other change should be well thought out. I just wanted to point this out.

**Comment by tuomas2** [comment]
Related: #1857

**Comment by tuomas2** [comment]
Name is about to change soon, see #1857 .

**Comment by meichthys** [comment]
I think we can close this now that the changes have been made(?)
